src/profile_manager/forms.py

```diff
@@ -28,9 +28,12 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-        self.fields['grad_year'] = forms.ChoiceField(
-            choices=Profile.get_grad_year_choices()
-        )
+        if SiteConfig.get().custom_profile_field:
+            self.fields['custom_profile_field'] = forms.ChoiceField(
+                choices=Profile.get_grad_year_choices()
```

Comment:
Per https://github.com/bytedeck/bytedeck/issues/1273#issuecomment-1364910592
"Remove any old references to Grad Year field in the code, including the default function."